core/templates/page_text.html

```diff
@@ -1,21 +1,23 @@
-{% extends "site_base.html" %}
-{% load static from staticfiles %}
+<!-- template for viewing the OCR version of pages, e.g.:
+        http://BASE_URL/lccn/sn99021999/1887-03-07/ed-1/seq-1/ocr/ -->
 
+{% extends "__l_main.html" %}
+{% load static from staticfiles %}
 {% block subcontent %}
 
 <div id="head_nav">
-    <p>
+    <h4>
     Below is the <a href="{% url 'openoni_ocr' %}">OCR text representation</a>
     for this newspapers page. It is also available as 
     <a href="{% url 'openoni_page_ocr_txt' title.lccn issue.date_issued issue.edition page.sequence %}">plain text</a> as well as 
     <a href="{% url 'openoni_page_ocr_xml' title.lccn issue.date_issued issue.edition page.sequence %}">XML</a>.
     </p>
```

Comment:
yup it should be an /h4. I was being slightly lazy here, it should probably be a [p class="whatever"] but I think an h4 is ok